src/window.rs

```diff
@@ -788,6 +788,9 @@ pub enum CursorIcon {
     NwseResize,
     ColResize,
     RowResize,
+
+    /// Represents a custom cursor which uses an image. At the current time, it's only implemented for Windows. The argument has to be a path pointing to a .cur or .ani file.
+    Custom(&'static str),
```

Comment:
Is there any particular reason we aren't using Winit's [`Icon`](https://docs.rs/winit/0.20.0-alpha3/winit/window/struct.Icon.html) type here? Even though the winapi docs for `SetCursor` don't explicitly say so, the function does accept the `HICON` handle provided by `platform_impl::windows::icon::WinIcon`, and it should make this variant easier to port between platforms.

Reply:
On X11 (and I imagine Windows is similar), cursor data may contain information that isn't represented by `Icon`. Specifically, it may contain a position (this is used to implement the crosshair, whose pointer position is in the center of the cursor image instead of the top left) and it may contain multiple images that are cycled through in an animation.

Reply:
We could change `Custom`'s signature to `Custom{ hotspot: PhysicalPosition, icon: Icon }` (or some other name for `hotspot` - that's the Windows terminology). WRT animated cursors - honestly, that seems like a niche enough feature that I don't think we need to officially support it, although API-wise we could use `Vec<Icon>`, `framerate: u32` if we really needed to.